notebooks/Analyze_Model_Outputs.ipynb

```diff
@@ -0,0 +1,3047 @@
+{
```

Comment:
<p>maybe also say that the line below uses <code>values</code>  to unwrap the TokenSpanArray from the Series?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good point; there's quite a bit here that I didn't explain. I've expanded on this cell significantly in the next commit.</p>